prisma/schema.prisma

```diff
@@ -7,27 +7,38 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
+enum CommentStatus {
+  pending
+  approved
+  rejected
```

Comment:
Using lowercase here as it is more common in TypeScript for enum-like values ([example 1](https://www.typescriptlang.org/docs/handbook/literal-types.html#string-literal-types), [example 2](https://www.typescriptlang.org/docs/handbook/unions-and-intersections.html#discriminating-unions)). Postgres is just an implementation detail of how we store data.